### ISSUE TITLE
Rabbit MQ 14:52:04.894 honeytrap/server dial tcp [::1]:5672: connect: connection refused

### ISSUE THREAD

**Issue by kounelios13** [honeytrap/honeytrap#490]
Hello . I am trying to use rabbit mq.
I have the folowing config file 
```toml
[listener]
type="socket"

[service.ssh-simulator]
type="ssh-simulator"
credentials=["root:root", "root:password"]

[[port]]
port="tcp/8022"
services=["ssh-simulator"]

[service.telnet]
type="telnet"
prompt=">>"
image="/data/screenshot-20171030174349.png"
server-name="WS-3244"

[[port]]
port="tcp/8023"
services=["telnet"]

[service.http01]
type="http"
server="Nginx"

[[port]]
port="tcp/8080"
services=["http01"]

[service.ftp]
type="ftp"

[[port]]
port="tcp/21"
services=["ftp"]

[service.adb01]
type="adb"

[[port]]
port="tcp/5037"
services=["adb01"]

[service.smtp]
type="smtp"

[[port]]
port="tcp/25"
services=["smtp"]

[channel.console]
type="console"

[channel.file]
type="file"
filename="/var/log/honeytrap.log"
maxsize=536870912
timeout="1h"

[channel.rabbitmq]
type="rabbitmq"
address="ampq://172.18.0.3:5762/"
queue="honeytrap"


[[filter]]
channel=["rabbitmq"]
service=["ssh"]

[[logging]]
output = "stdout"
level = "debug"

```

However when I try to start honeytrap I get this error:
`14:52:04.894 honeytrap/server ▶ CRIT 004 Error initializing channel rabbitmq(rabbitmq): dial tcp [::1]:5672: connect: connection refused
`
However If i try to connect to the same rqbbit mq server from another program it works fine.Any Ideas what I am doing wrong?

**Comment by nl5887** [comment]
#491 should fix this issue, could you verify?

**Comment by kounelios13** [comment]
@nl5887 when I try to pull master builf fails with this error
```
# github.com/google/netstack/tcpip/link/rawfile
../../../go/pkg/mod/github.com/honeytrap/netstack@v0.0.0-20190414201528-9ea5e4d2258f/tcpip/link/rawfile/rawfile_unsafe.go:126:10: undefined: BlockingPoll
../../../go/pkg/mod/github.com/honeytrap/netstack@v0.0.0-20190414201528-9ea5e4d2258f/tcpip/link/rawfile/rawfile_unsafe.go:148:10: undefined: BlockingPoll
../../../go/pkg/mod/github.com/honeytrap/netstack@v0.0.0-20190414201528-9ea5e4d2258f/tcpip/link/rawfile/rawfile_unsafe.go:178:14: undefined: BlockingPoll
# github.com/google/netstack/tcpip/stack
../../../go/pkg/mod/github.com/honeytrap/netstack@v0.0.0-20190414201528-9ea5e4d2258f/tcpip/stack/stack.go:342:12: undefined: tcpip.StdClock

```

**Comment by kounelios13** [comment]
Truth is that the first time I just cloned the source tree from this: #488 to be able to compile honeytrap.Maybe it should me merged into master first

**Comment by kounelios13** [comment]
So after messing up a little bit I managed to do the following
* Merge `netstack-to-gvisor` from #488 to master
* Build the project

After that I removed the port from the uri and it worked

**Comment by sammynx** [comment]
Nice you got it to work. Indeed #488 needs to be merged first to get it to work.

Thanks for your input.

**Comment by kounelios13** [comment]
You are welcome.I hope it gets merged soon :)
